PaymentHighway/Extensions.swift

```diff
@@ -96,21 +95,26 @@ internal extension UIColor {
 
 public extension UIViewController {
 	
-    @objc public func presentSPHAddCardViewController(_ source: UIViewController, animated: Bool, transactionId : String, success: @escaping (String) -> (), error: @escaping (NSError) -> (), completion: (() -> Void)?) {
+    @objc public func presentSPHAddCardViewController(_ source: UIViewController,
+                                                      animated: Bool,
+                                                      transactionId : String,
+                                                      success: @escaping (String) -> Void,
+                                                      error: @escaping (NSError) -> Void,
+                                                      completion: (() -> Void)?) {
 		let storyboard = UIStoryboard(name: "SPH", bundle: Bundle(for: SPH.self))
-		let controller = storyboard.instantiateViewController(withIdentifier: "SPHAddCardForm") as! SPHAddCardViewController
+        guard let controller = storyboard.instantiateViewController(withIdentifier: "SPHAddCardForm") as? SPHAddCardViewController else { return }
```

Comment:
This fails silently now also, which is not optimal. I don't think every forced cast should be removed just because the linter says so. Both silent fail and crash result in the same poor user experience, but at least with a crash you get to fix the issue faster (and also get stack traces from crash logs).

Reply:
Personally a crash from the end user is worst that a non action but agree that both are bad user experience. As a general comment i do not like the use of force casting or force unwrapping , even tough even linter said the same. in this specific case will never happen a crash and we can keep the force casting.